src/components/ResourceList/ResourceList.tsx

```diff
@@ -329,13 +329,15 @@ class ResourceList extends React.Component<CombinedProps, State> {
     };
   }
 
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
ESLint wants us to put private methods below public ones like render(). However in this file that'll make the diff incredibly noisy so I've surpassed the warning